### PR TITLE
Feat/gh12 hub session header

### PR DESCRIPTION
**Pull request by nathanatstariongroup**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/DEH-MDSYSML/pulls) open
- [x] I have verified that I am following the DEH-CommonJ [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/DEH-CommonJ/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
fixes RHEAGROUP/DEH-MDSYSML#12

- [x] Added Session Header inormation

### CODE REVIEW

File: src/main/java/App/AppContainer.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+ * AppController.java
+ *
+ * Copyright (c) 2020-2021 RHEA System S.A.
+ *
+ * Author: Sam Geren�, Alex Vorobiev, Nathanael Smiechowski 
+ *
+ * This file is part of DEH-CommonJ
+ *
+ * The DEH-MDSYSML is free software; you can redistribute it and/or
```
Comment (samatrhea) [inline src/main/java/App/AppContainer.java:10]: incorrect project name

**Review by samatrhea — CHANGES_REQUESTED** [review]
did some spot checks, can you have a look at all headers. they reference the sysml project, not the common project.

File: src/main/java/Views/HubLogin.java [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+                        .forEachOrdered(x -> this.engineeringModelSetupComboBox.addItem(x));
+                }
+                else
+                {
+                    JOptionPane.showMessageDialog(this.loginPanelContainer, "An error occured while trying to login. Have a look at the Cameo/MagicDraw log for detail.");
+                }
+            }
+            else
+            {
+                JOptionPane.showMessageDialog(this.loginPanelContainer, "Please select an valid server address to continue");
```
Comment (arielleatrhea) [inline src/main/java/Views/HubLogin.java:475]: "a valid server address"

**Review by arielleatrhea — APPROVED** [review]
Seems good

**Review by samatrhea — APPROVED** [review]

**Pull request merged**